qiskit/algorithms/time_evolution/imaginary/qite.py

```diff
@@ -0,0 +1,21 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Interface for Quantum Imaginary Time Evolution."""
+
+from abc import ABC
+
+from qiskit.algorithms.time_evolution.evolution_base import EvolutionBase
+
+
+class Qite(EvolutionBase, ABC):
```

Comment:
I have no horse in this race, but if you'd like to stay PEP8-compliant, then acronyms are usually written in all caps.